cocos/particle/renderer/particle-system-renderer-cpu.ts

```diff
@@ -642,15 +642,17 @@ export default class ParticleSystemRendererCPU extends ParticleSystemRendererBas
         }
 
         if (ps.sharedMaterial == null && this._defaultMat == null) {
-            _matInsInfo.parent = builtinResMgr.get<Material>('default-particle-material');
-            _matInsInfo.owner = this._particleSystem;
-            _matInsInfo.subModelIdx = 0;
-            this._defaultMat = new MaterialInstance(_matInsInfo);
-            _matInsInfo.parent = null!;
-            _matInsInfo.owner = null!;
-            _matInsInfo.subModelIdx = 0;
-            if (this._renderInfo!.mainTexture !== null) {
-                this._defaultMat.setProperty('mainTexture', this._renderInfo!.mainTexture);
+            if (Editor) {
```

Comment:
I don't think it can be changed that way, the user may not have any materials in the old project, the previous version worked, but now it doesn't work.